executor/ddl.go

```diff
@@ -371,8 +432,24 @@ func (e *DDLExec) executeDropSequence(s *ast.DropSequenceStmt) error {
 
 // dropTableObject actually applies to `tableObject`, `viewObject` and `sequenceObject`.
 func (e *DDLExec) dropTableObject(objects []*ast.TableName, obt objectType, ifExists bool) error {
-	var notExistTables []string
```

Comment:
You can remove all  all temporary tables from `DropTableStmt`, then dropTableObject need not to be modified.